Rubberduck.CodeAnalysis/Inspections/Concrete/ImplementedInterfaceMemberInspection.cs

```diff
@@ -40,49 +40,57 @@ namespace Rubberduck.CodeAnalysis.Inspections.Concrete
         {}
 
-        protected override bool IsResultDeclaration(Declaration declaration, DeclarationFinder finder)
+        protected override (bool, IEnumerable<ModuleBodyElementDeclaration>) IsResultDeclarationWithAdditionalProperties(Declaration declaration, DeclarationFinder finder)
         {
-            if (!IsInterfaceDeclaration(declaration))
+            if (!(declaration is ClassModuleDeclaration classModule && classModule.IsInterface)) 
             {
-                return false;
+                return (false, null);
             }
 
+            var implementedMembers = HasImplementedMembers(declaration, finder);
+            return (implementedMembers.Count() > 0, implementedMembers);
+        }
+
+        private IEnumerable<ModuleBodyElementDeclaration> HasImplementedMembers(Declaration declaration, DeclarationFinder finder) 
+        {
             var moduleBodyElements = finder.Members(declaration, DeclarationType.Member)
                 .OfType<ModuleBodyElementDeclaration>();
 
             return moduleBodyElements
-                .Any(member => member.Block.ContainsExecutableStatements(true));
-        }
-
-        private static bool IsInterfaceDeclaration(Declaration declaration)
-        {
-            if (!(declaration is ClassModuleDeclaration classModule))
-            {
-                return false;
-            }
-            return classModule.IsInterface
-                || declaration.Annotations.Any(an => an.Annotation is InterfaceAnnotation);
+                .Where(member => member.Block.ContainsExecutableStatements(true));
         }
 
-        protected override string ResultDescription(Declaration declaration)
+        protected override string ResultDescription(Declaration declaration, IEnumerable<ModuleBodyElementDeclaration> results)
         {
-            var qualifiedName = declaration.QualifiedModuleName.ToString();
-            var declarationType = Resources.RubberduckUI.ResourceManager
-                .GetString("DeclarationType_" + declaration.DeclarationType)
-                .Capitalize();
             var identifierName = declaration.IdentifierName;
+            var memberResultsString = FormatResults(results);
 
             return string.Format(
                 InspectionResults.ImplementedInterfaceMemberInspection,
-                qualifiedName,
-                declarationType,
-                identifierName);
+                identifierName,
+                memberResultsString);
+        }
+
+        private static string FormatResults(IEnumerable<ModuleBodyElementDeclaration> results)
+        {
+            List<string> items = new List<string>();
+
+            foreach (ModuleBodyElementDeclaration elem in results)
+            {
+                var memberType = Resources.RubberduckUI.ResourceManager
+                    .GetString("DeclarationType_" + elem.DeclarationType)
+                    .Capitalize();
```

Comment:
Why `Capitalize`? I am not 100% sure whether this will honor our language settings.

Reply:
This is what was in the code before I edited anything.